builder/src/test/java/com/iluwatar/builder/HeroTest.java

```diff
@@ -35,6 +39,61 @@
  */
 public class HeroTest {
 
+  @BeforeAll
+  public static void setupCoverage() {
+    Hero.coverage = new boolean[13];
+    Hero.totalCoverage = new boolean[13];
+  }
+
+  /**
+   * Print coverage and empty array
+   */
+  @AfterEach
+  public void printAndCleanCoverage() {
+    // If default is false, method has not been run, don't print
+    if (Hero.coverage[12] == false) {
+      return;
+    }
+
+    System.out.print("[");
+    for (int i = 0; i < Hero.coverage.length; i++) {
+      if (Hero.coverage[i] || Hero.totalCoverage[i]) {
+        Hero.totalCoverage[i] = true;
+      }
+      if (i < Hero.coverage.length - 1) {
+        System.out.print(Hero.CONDS[i] + " is " + Hero.coverage[i] + ", ");
+      } else if (i == Hero.coverage.length - 1) {
+        System.out.print(Hero.CONDS[i] + " is " + Hero.coverage[i]);
+      }
+      Hero.coverage[i] = false;
```

Comment:
Unnecessary step since you set the array to false at row 71

Reply:
Good spotted, have removed it now!